Source/Csla/Serialization/Mobile/TypeMap.cs

```diff
@@ -0,0 +1,38 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="TypeMap.cs" company="Marimer LLC">
+//     Copyright (c) Marimer LLC. All rights reserved.
+//     Website: https://cslanet.com
+// </copyright>
+// <summary>Maps a type to a serializer type.</summary>
+//-----------------------------------------------------------------------
+
+namespace Csla.Serialization.Mobile;
+
+/// <inheritdoc />
+public class TypeMap<T, S> : ITypeMap
+  where S : IMobileSerializer
+{
+  /// <summary>
+  /// Creates an instance of the class.
+  /// </summary>
+  public TypeMap()
+  {
+    CanSerialize = (t) => t == OriginalType;
```

Comment:
This ctor should chain to the other one.

Reply:
yes